src/eko/couplings.py

```diff
@@ -46,12 +46,12 @@ def expanded_lo(ref, beta0, lmu):
         beta0: float
             first coefficient of the beta function
         lmu: float
-            log(scale_to / scale_from)
+            scaling log
 
     Returns
     -------
         a : float
-            coupling at target scale :math:`a(scale_to^2)`
+            coupling at target scale :math:`a(µ_r^2)`
```

Comment:
Please don't follow me blindly :see_no_evil: use your own brain :brain:  I meant this more in a "global" sense - follow the notation which is already there: e.g. https://eko.readthedocs.io/en/latest/theory/pQCD.html you see the default notation is `\mu_R`; same with the "scaling log" - you know what it is and if ever you find a better name or explanation (like ratio between reference scale and target scale or log expansion parameter or ...) use it - us your physics intuition :rocket: ;-)